RSI

```diff
@@ -15,8 +13,21 @@ start_time = datetime.datetime(2020, 10, 1)
 #end_time = datetime.datetime(2020, 1, 20)
 end_time = datetime.datetime.now().date().isoformat()         # today
 
-df = ticker_df
+connected = False
+while not connected:
+    try:
+        ticker_df = web.get_data_yahoo(ticker, start=start_time, end=end_time)
+        connected = True
+        print('connected to yahoo')
+    except Exception as e:
+        print("type error: " + str(e))
+        time.sleep( 5 )
+        pass
```

Comment:
ticker_df = web.get_data_yahoo(ticker, start=start_time, end=end_time)

reliable API, no need to declare connectivity